src/common/net/index.ts

```diff
@@ -1,19 +1,10 @@
 import { FetchHttpClient } from './fetch-client';
 import { HttpClient } from './http-client';
-import { NodeHttpClient } from './node-client';
 
 export function createHttpClient(
```

Comment:
I like leaving `createHttpClient` in place for now to make the code change easier, but maybe later we can inline `new FetchHttpClient` into the places we are calling `createHttpClient` and get rid of that layer of indirection.

Reply:
Wait, do we even call this anywhere? I see down below we had this same logic under `WorkOSNode.createHttpClient` method. 🤔

In any case, something that can be cleaned up later.

Reply:
Yeah, you're right. This file isn't used at all and can be removed.